package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "erlang_ls",
+    "version": "0.0.1",
+    "description": "Erlang Language Server",
+    "homepage": "https://github.com/erlang-ls/vscode#readme",
```

Comment:
Maybe point to the `erlang_ls` repo (especially below in the `bugs` section)? I guess that's where most people will open issues.

Reply:
I've changed the issues link to point at the `erlang_ls` repo. The other links I think should take users to the extension itself unless you see any problems with that?

Reply:
In retrospection, it's probably fine to point to the vscode repo as long as the description clarifies this refers to the VSCode extension and not to the "Erlang Language Server". I will take care of that.